test/screenshot/spec/mdc-top-app-bar/classes/baseline-dense.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
IE11 screenshots are not loading, any idea?

https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/08/23_32_31_196/report/report.html

Reply:
Maybe retry? What do you think?

Reply:
Fixed in retry